src/system.js

```diff
@@ -14,16 +14,25 @@ contexts.forEach(context => {
   context.keys().forEach(key => components.push(context(key).default))
 })
 
+function initializeCustomProps(tokens, prefix) {
+  if (typeof tokens === "undefined" || Object.keys(tokens).length === 0) {
```

Comment:
```suggestion
  if (!tokens) {
```
should be sufficient

Reply:
Love it!

This only works for nonexistent tokens, if the token object is empty (`spacing: {}`) it doesn't return early but loops for...no content - whatever...

Reply:
yep, looping on empty input is fine :-)